clients/web/src/utilities/EventStream.js

```diff
@@ -64,6 +73,7 @@ prototype.open = function () {
                 stream.trigger('g:error', e);
                 return;
             }
+            window.localStorage.setItem('sseTimestamp', obj._girderTime);
```

Comment:
My latest commits catches errors in the localstore API and falls back on an "in memory" timestamp.  Local storage is pretty ubiquitous these days, I don't think it is worth using a cookie, but if someone feels strongly about it, I can make the switch.